core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -64,6 +64,8 @@ private[spark] class TaskSchedulerImpl(
   // How often to check for speculative tasks
   val SPECULATION_INTERVAL_MS = conf.getTimeAsMs("spark.speculation.interval", "100ms")
 
+  val MIN_TIME_TO_SPECULATION = 100
```

Comment:
Can you add a comment here? Something like "Duplicate copies of a task will only be launched if the original copy has been running for at least this amount of time. This is to avoid the overhead of launching speculative copies of tasks that are very short."